src/SketchField.jsx

```diff
@@ -15,6 +15,13 @@ import Tool from './tools';
 
 const fabric = require('fabric').fabric;
 
+const styles = {
```

Comment:
Couldn't do it with `wrapperEl.style` because it is a `StaticCanvas` (another type of Fabric canvas)

Reply:
Are you tried inspecting the element? Maybe there is an option to avoid create this object

Reply:
Done